package.json

```diff
@@ -27,7 +28,6 @@
     "eslint-plugin-node": "^5.1.1",
     "eslint-plugin-prettier": "^2.2.0",
     "eslint-plugin-promise": "^3.5.0",
-    "prettier": "^1.5.3",
-    "prettier-eslint": "^6.4.2"
+    "prettier": "^1.5.3"
```

Comment:
1.6.0 ?

Reply:
I guess the next issue to fix is #25 then :P

Reply:
What I meant is that when you update to prettier it will change a bunch of things as well.

Reply:
What do you mean? I was already using prettier, and I'm still using it. I'm just not using prettier-eslint anymore (which actually wasn't used... I just removed the useless requirement)